src/pages/RFOX/hooks/useEpochHistoryQuery.ts

```diff
@@ -12,9 +12,13 @@ import {
 
 type EpochHistoryQueryKey = ['epochHistory']
 
-// TODO(gomes): revert me -  this obviously won't work until first rewards epoch start
+// TODO: Clean up by removing the Math.min after the first epoch starts.
+// This is a temporary hack to ensure we have an epoch to test with prior to rFOX launch,
+// and the correct one after launch (in case we don't action this todo it for any reason).
 // const RFOX_FIRST_EPOCH_START_TIMESTAMP = BigInt(dayjs('2024-07-01T00:00:00Z').unix())
-const RFOX_FIRST_EPOCH_START_TIMESTAMP = BigInt(dayjs().subtract(1, 'month').unix())
+const RFOX_FIRST_EPOCH_START_TIMESTAMP = BigInt(
+  Math.min(dayjs().subtract(1, 'month').unix(), dayjs('2024-07-01T00:00:00Z').unix()),
```

Comment:
Can we use `30 days` here? see https://github.com/shapeshift/web/pull/7204 where everything has been moved to 30 days instead of 1 month

Reply:
As discussed, will keep aligned to calendar month.

For posterity, epochs are aligned to calendar month, UTC:
https://forum.shapeshift.com/t/ideation-scp-166-thorfox-a-fox-tokenomics-upgrade-proposal/2932/51?u=0xean
https://github.com/shapeshift/rFOX/wiki/rFOX#what-is-rfox

Reply:
Discussed this one out, I'm a derp and `1 month` is the *more* semantically correct one (`30 days` is absolutely wrong, disregard me), but shifting to the start of next month vs. adding a month is the goer!